doc/guides/using-internal-errors.md

```diff
@@ -15,8 +15,8 @@ to be breaking changes. By providing a permanent identifier for a specific
 error, we reduce the need for userland code to inspect error messages.
 
 *Note*: Switching an existing error to use the `internal/errors` module must be
-considered a `semver-major` change. However, once using `internal/errors`,
-changes to `internal/errors` error messages will be handled as `semver-minor`
+considered a `semver-major` change. In the future, after the transition
+has been completed, error message changes should be handled as `semver-minor`
```

Comment:
Nit: This makes it sound like once the errors have been all migrated, the change to treating them as not-breaking is automatic. Honestly, I think the best/easiest thing to do at this point is to remove the sentence. It's not needed. It's about a possible future state that is likely to come (but not guaranteed) and we certainly haven't agreed as to exactly when it will come. So just leave it out.